scripts/core/menu/menu.ts

```diff
@@ -142,11 +142,7 @@ angular.module('superdesk.core.menu', [
                 api.query('backend_meta', {}).then(
                     (metadata) => {
                         scope.build_rev = metadata.meta_rev;
-                        scope.build_rev_core = metadata.meta_rev_core;
-                        scope.build_rev_client = metadata.meta_rev_client;
-                        scope.build_rev_planning = metadata.meta_rev_planning;
-                        scope.build_rev_publisher = metadata.meta_rev_publisher;
-                        scope.build_rev_analytics = metadata.meta_rev_analytics;
+                        scope.modules = metadata.modules.filter((mod) => mod !== null);
```

Comment:
yep

Reply:
:D why?

Reply:
I don't know :wink: will avoid it

Reply:
How do you know there can be nulls? Did you see one during runtime? If someone is careless enough to try to register a `null` as a plugin, we should throw an error when registering it instead of accepting it as a valid option and handling it everywhere in the app.